horovod/common/controller.cc

```diff
@@ -449,6 +449,36 @@ Response Controller::ConstructResponse(std::string& name, int joined_size) {
     }
   }
 
+  // If we are doing an allreduce, check that prescaling and postscaling factors
+  // are identical.
```

Comment:
For clarity, can you change this to "are identical across ranks"?

Reply:
Yep, no problem. Done.